src/parlant/core/engines/alpha/prompt_builder.py

```diff
@@ -238,3 +240,24 @@ def add_staged_events(
             )
 
         return self
+
+    def add_few_shots(
```

Comment:
I don't think we need this. Let @MCBarKar format things however we wishes for each component, manually.
This design is also not great since it ties presentation logic into the shot itself via the `format` method.